docs_versioned_docs/version-ros2humble/ros/networking/network_troubleshooting/intermittent_connectivity.mdx

```diff
@@ -0,0 +1,175 @@
+---
+title: Intermittent Connectivity
+sidebar_position: 4
+---
+
+Intermittent connectivity, where communication drops in and out or slows down, can be challenging to diagnose. Unlike issues that entirely block communication, inconsistent communication arises from transient or situational factors, which often involve complex interactions between multiple factors. This page reviews several common causes and how to address them.
+
+## Wi-Fi Interference or Low Signal Strength {#weak-wi-fi}
+Wireless networks can be significantly impacted by many factors such as interference or weak signal, resulting in packet loss and fluctuating bandwidth. Within the context of ROS, this can manifest as lost messages, delayed messages or nodes disappearing intermittently as a few examples.
+
+There are two ways to approach this issue. The first is to improve the network capacity so that it can support more data transmission. This should be considered as the first approach to fixing the issue as there can often be some quick fixes that provide measurable improvement. Secondary to that, the amount of data being transmitted should be reduced as is necessary to bring it into alignment with the abilities of the network. This second part is covered under the [Insufficient Bandwidth](#insufficient-bandwidth) section.
+
+To evaluate the network, it is best to start by stopping all ROS systems on the network and then using [iperf3](network_troubleshooting_tools.mdx#iperf3) to verify the network bandwidth. This should be done with the robot in several locations, particularly any location where the instability issues occur. In general, as a rule of thumb, if the bandwidth is less than 40 Mbps then it is a low bandwidth network and it must be very carefully managed to support a ROS 2 system.
```

Comment:
`stopping all ROS systems on` consider using `ROS 2` as that seems be be commonly used.